stock_reorder_limit/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Sensible limits on minimum stock rule processing",
```

Comment:
HI @NL66278 thanks for your good contribution.
Just an idea. I don't want block the merge but I imagine that technical module name and name key content are so different than it could be confusing. Don't you think ? 
Note your name key content here fit perfectly for summary content which is also searchable in apps.odoo.com